pkg/middleware/create_home.go

```diff
@@ -16,7 +19,41 @@ func CreateHome(opts ...Option) func(next http.Handler) http.Handler {
 
 	return func(next http.Handler) http.Handler {
 		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			accounts := opt.AccountsClient
+
+			tokenManager, err := jwt.New(map[string]interface{}{
+				"secret": opt.TokenManagerConfig.JWTSecret,
+			})
+			if err != nil {
+				w.WriteHeader(http.StatusInternalServerError)
+				return
+			}
+
 			token := r.Header.Get("x-access-token")
+			if token == "" {
```

Comment:
do we want to halt every other middleware if the token is not present? Or do we also need to call `next.ServeHTTP(w, r)` before return?

Reply:
Fixed